code/cfile/cfile.cpp

```diff
@@ -863,7 +876,12 @@ int cfget_cfile_block()
 
 	// If we've reached this point, a free Cfile_block could not be found
 	nprintf(("Warning","A free Cfile_block could not be found.\n"));
-	Assert(0);	// out of free cfile blocks
+
+	// Dump a list of all opened files
+	mprintf(("Out of cfile blocks! Currently opened files:\n"));
+
```

Comment:
:D I only noticed because I'm using this to track down a cfile leak; nice work on the rest of the code; very useful